website/content/docs/secrets/databases/elasticdb.mdx

```diff
@@ -130,6 +130,16 @@ Now, Elasticsearch is configured and ready to be used with Vault.
        client_key=$ES_HOME/config/certs/elastic-certificates.key.pem
    ```
 
+## Usage
+
+After the secrets engine is configured and a user/machine has a Vault token with
+the proper permission, it can generate or rotate credentials.
```

Comment:
```suggestion
Once you configure the secrets engine, Vault will automatically generate and rotate credentials when users and machines connect using a Vault token with the proper permissions.
```

Style correction: make it clear who/what takes action (although I'm not sure I interpreted what you were trying to say correctly)

Reply:
Hey @schavis , this section was moved from below and is what is already published, so I'm not sure what the original implementor meant.

I can give a bit more context on how the flow works so we can find a better way to document it.

Using a Vault database plugin goes through 3 "steps":
* Registering the plugin, typically done by an operator (only required if it is not pre-bundled, so not necessary here for elasticsearch)
* Setup the plugin, typically done by an operator, which is done by 
  * Enabling the plugin
  * Passing a configuration so Vault can authenticate with the external system
  * Configuring roles which will be used to generate user and passwords on the external system
* Get credentials from the plugin, done by a client (user or machine)
  * Authenticate to Vault
  * Read credentials for the roles that were defined earlier

I think the intention here is a transitory statement to distinguish between the action of configuring the plugin versus configuring the roles.

Reading examples from other Database engines, it seems like many of them use:
```
After the secrets engine is configured, configure dynamic and static roles to enable generating credentials.
```
Would it be ok to use the same phrase here?